### PR TITLE
Fix #7914: When I start my questionnaire and go back to start and then start again, my questionnaire does not start

### LINKED ISSUE #7914: When I start my questionnaire and go back to start and then start again, my questionnaire does not start
### How to Reproduce
Upload a questionnaire
Set row level security for questionnaire
Go to Questionnaire plugin
Click View Questionnaire
Click Start Questionnaire
Fill something in
Click Go back to start
Click Start Questionnaire

### Expected behavior
Questionnaire loads again

### Observed behavior
Loading and nothing happens

### PR DESCRIPTION
**Pull request by sidohaakma**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: molgenis-questionnaires/src/main/frontend/src/store/actions.js [hunk 1]
```diff
@@ -69,6 +69,8 @@ const actions = {
           handleError(commit, error)
           reject(error)
         })
+      } else {
+        resolve()
       }
 
       commit('SET_LOADING', false)
```
Comment (ConnorStroomberg) [inline molgenis-questionnaires/src/main/frontend/src/store/actions.js]: move this line into the else ?

File: molgenis-questionnaires/src/main/frontend/test/unit/specs/store/actions.spec.js [hunk 2]
```diff
@@ -283,6 +283,22 @@ describe('actions', () => {
       testAction(actions.GET_QUESTIONNAIRE, questionnaireId, state, expectedMutations, [], done)
     })
 
+    it('should do nothing, only resolve the promise when questionnaire has already been retrieved', done => {
+      const questionnaireId = 'test_quest'
+
+      const state = {
+        username: 'testuser'
```
Comment (ConnorStroomberg) [inline molgenis-questionnaires/src/main/frontend/test/unit/specs/store/actions.spec.js]: is this needed ?

File: molgenis-questionnaires/src/main/frontend/src/store/actions.js (hunk 1 above)
Comment (sidohaakma) [reply molgenis-questionnaires/src/main/frontend/src/store/actions.js]: done

File: molgenis-questionnaires/src/main/frontend/test/unit/specs/store/actions.spec.js (hunk 2 above)
Comment (sidohaakma) [reply molgenis-questionnaires/src/main/frontend/test/unit/specs/store/actions.spec.js]: yes

**Pull request merged**